app/src/main/java/com/infomaniak/swisstransfer/ui/screen/newtransfer/importfiles/components/TransferTypeButtons.kt

```diff
@@ -18,93 +18,53 @@
 package com.infomaniak.swisstransfer.ui.screen.newtransfer.importfiles.components
 
 import android.content.res.Configuration
-import androidx.compose.foundation.layout.*
-import androidx.compose.foundation.lazy.grid.GridCells
-import androidx.compose.foundation.lazy.grid.LazyVerticalGrid
-import androidx.compose.foundation.lazy.grid.items
-import androidx.compose.material3.*
+import androidx.annotation.StringRes
+import androidx.compose.foundation.horizontalScroll
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.material3.Surface
 import androidx.compose.runtime.Composable
-import androidx.compose.ui.Alignment
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.saveable.rememberSaveable
 import androidx.compose.ui.Modifier
-import androidx.compose.ui.res.stringResource
-import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.graphics.vector.ImageVector
 import androidx.compose.ui.tooling.preview.Preview
-import androidx.compose.ui.unit.dp
-import com.infomaniak.swisstransfer.ui.screen.newtransfer.importfiles.TransferType
+import com.infomaniak.swisstransfer.R
+import com.infomaniak.swisstransfer.ui.images.AppImages.AppIcons
+import com.infomaniak.swisstransfer.ui.images.icons.Chain
+import com.infomaniak.swisstransfer.ui.images.icons.Envelope
+import com.infomaniak.swisstransfer.ui.images.icons.QrCode
+import com.infomaniak.swisstransfer.ui.images.icons.WifiWave
+import com.infomaniak.swisstransfer.ui.theme.Margin
 import com.infomaniak.swisstransfer.ui.theme.SwissTransferTheme
 
-private val SPACE_AROUND_BUTTONS = 20.dp
-private val COLUMN_MIN_WIDTH = 150.dp
-
 @Composable
-fun TransferTypeButtons(items: List<TransferType>, navigateToTransfer: (TransferType) -> Unit) {
-    LazyVerticalGrid(
+fun TransferTypeButtons(selectedTransferType: TransferType) {
```

Comment:
Maybe `selectedTransferType` could be something like `defaultSelectedTransferType` or similar? Or maybe even "initial"